### PR TITLE
Fixes #1745: Add a (dumb) Milestone editor widget

### LINKED ISSUE #1745: Create Milestones editor widget
Basically copy the existing label editor and stick it below.

We can create a dumb UI before having it do anything, or just read-only.

#886.

### PR DESCRIPTION
**Pull request by miketaylr**
It doesn't read real milestone data yet, and it still can't save or change milestones. 

r? @MDTsai

### CODE REVIEW

File: webcompat/static/js/lib/editor.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+  }
+});
+
+/* Child classes need to define the following methods/properties, or this will explode:
+    * closeEditor
+    * template
+    * updateView
+*/
+issues.CategoryEditorView = Backbone.View.extend({
+  // todo, change these classes to be itemeditor, etc.
```
Comment (miketaylr) [inline webcompat/static/js/lib/editor.js]: nit: remove this comment

File: webcompat/templates/issue/issue-milestones.jst [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<!-- note: it might not make sense to get this data from the model... i need currently selected milestone, which i already have -->
```
Comment (miketaylr) [inline webcompat/templates/issue/issue-milestones.jst]: nit: remove this comment

File: webcompat/static/js/lib/milestones.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+      issueView: this
+    });
+    if (this._isLoggedIn) {
+      //this.issueLabels = this.getIssueMilestones();
+      this.editorButton.show();
+    }
+  },
+  getIssueMilestones: function() {
+    // TODO: have this do the right thing...
+    return _.pluck(this.model.get("labels"), "name");
```
Comment (MDTsai) [inline webcompat/static/js/lib/milestones.js]: this.model.get("milestones") ?

**Comment by miketaylr** [comment]
Thanks!

File: webcompat/static/js/lib/milestones.js (hunk 3 above)
Comment (miketaylr) [reply webcompat/static/js/lib/milestones.js]: Yeah, I have a follow up issue for that. #1766

**Pull request merged**